server/src/main/java/io/crate/expression/symbol/Symbols.java

```diff
@@ -137,6 +102,19 @@ public static void toStream(Collection<? extends Symbol> symbols, StreamOutput o
         }
     }
 
+    public static List<Symbol> listFromStream(StreamInput in) throws IOException {
+        return in.readList(Symbols::fromStream);
+    }
+
+    @Nullable
+    public static Symbol nullableFromStream(StreamInput in) throws IOException {
```

Comment:
Single value streamer functions are still here - should I move these too? Would kinda have to going by the PR descriptions split decision

Reply:
You mean to move them to `Symbol`?

Reply:
yes

Reply:
Yep makes sense to do.